codegen/src/types.rs

```diff
@@ -1203,7 +1267,7 @@ impl FileDescriptor {
         writeln!(w, "#![allow(non_snake_case)]")?;
         writeln!(w, "#![allow(non_upper_case_globals)]")?;
         writeln!(w, "#![allow(non_camel_case_types)]")?;
-
+        writeln!(w, "#![allow(unused_imports)]")?;
```

Comment:
I believe we will open an issue later to find out if we can remove some of these attributes automatically in the future. Let's not block the PR